.gitignore

```diff
@@ -83,3 +83,6 @@ target/
 
 # pycharm
 .idea/
+
```

Comment:
Can you remove these lines from the `.gitignore` please?

Reply:
Thought it won't hurt for people using vscode, but alright.